scripts/generate_markdown_readme.py

```diff
@@ -20,6 +23,35 @@
         raise SystemExit(1)
     raise (e)
 
+def process_commands(wdl_file):
+    dir_name = os.path.dirname(args.input_wdl_path)
```

Comment:
replace args.input_wdl_path with wdl_file from the function definition

Reply:
fixed!